### ISSUE TITLE
(Classification approach) Last class gives always empty exposure

### ISSUE THREAD

**Issue by matamadio** [GFDRR/CCDR-tools#4]
No matter the number of bins (classes) specified, the last one will always produce zero or null output.

**Comment by ConnectedSystems** [comment]
I'm seeing outputs for Class 6 (the last class):

<img width="476" alt="image" src="https://user-images.githubusercontent.com/4075136/156868006-1e84d81d-bf4a-4429-9d9c-c7c41e7e7ef2.png">


Or do you mean something else?

**Comment by matamadio** [comment]
This happens in the landslide notebook I tried to do, setting a smaller number of classes.

```
class_edges = OrderedDict({
    f'class_{i}': widgets.BoundedFloatText(
        value=k,
        min=0.0,
        max=10.0,
        step=0.0001,
        description=f'Class {i}:',
        tooltip=f'Minimum value of class {i}. Value must be less than the next entry.',
        disabled=True
    ) for (i,k) in zip(range(1,5), [0.0001, 0.001, 0.01, 0.05])
})
```
Am I doing it wrong? :(
Maybe safer if you stem out the class-only versions, and I will then edit parameters for each hazard:
- Classes only, no RP loop - single hazard raster (for landslide and drought) 
- Classes only, with RP loop - just remove the "impact function" loop and choice (for heat stress and wildfires)

**Comment by matamadio** [comment]
Here is the bugged [landslide notebook](https://github.com/GFDRR/CCDR-tools/blob/sandbox/Top-down/notebooks/Landslide.ipynb).

**Comment by ConnectedSystems** [comment]
Fixed in commit 957517d451853c8442aedb07fc0d5c10bc1f2abb
